novawalletTests/Modules/Settings/SettingsTests.swift

```diff
@@ -77,6 +77,15 @@ final class SettingsTests: XCTestCase {
                 completion(nil)
             }
         }
+        
+        let operationQueue = OperationManagerFacade.sharedDefaultQueue
+        let web3AlertsSyncService = Web3AlertsSyncServiceFactory(
```

Comment:
It is assumed that unit tests doesn't make any remote requests or communicate with local database because this will lead to not deterministic behaviour.  So, let's pass stubs for web3alert and push notification services.